### PR TITLE
Allow mongodb

### PR DESCRIPTION
**Pull request by core23**
## Subject
This is needed for https://github.com/symfony/recipes-contrib/pull/622

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Added
- Added support for mongo document user
```

### CODE REVIEW

**Comment by OskarStark** [comment]
Please check Travis

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -69,6 +69,8 @@ public function getConfigTreeBuilder()
                         ->scalarNode('action_component')->defaultValue('%spy_timeline.class.action_component%')->cannotBeEmpty()->end() // fix the actionComponent deprecated parameter ...
                         ->scalarNode('action')->defaultValue('%spy_timeline.class.action%')->cannotBeEmpty()->end()
                         ->scalarNode('timeline')->defaultValue('%spy_timeline.class.timeline%')->cannotBeEmpty()->end()
+                        // NEXT_MAJOR: Change this to App\Enity\User
```
Comment (greg0ire) [inline src/DependencyInjection/Configuration.php]:
```suggestion
                        // NEXT_MAJOR: Change this to App\Entity\User
```

File: src/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -65,10 +65,12 @@ public function getConfigTreeBuilder()
                     ->end()
                     ->children()
                         ->scalarNode('component')->defaultValue('%spy_timeline.class.component%')->cannotBeEmpty()->end()
-                        ->scalarNode('actionComponent')->end()
+                        ->scalarNode('actionComponent')->setDeprecated('Use action_component node instead')->end()
                         ->scalarNode('action_component')->defaultValue('%spy_timeline.class.action_component%')->cannotBeEmpty()->end() // fix the actionComponent deprecated parameter ...
                         ->scalarNode('action')->defaultValue('%spy_timeline.class.action%')->cannotBeEmpty()->end()
                         ->scalarNode('timeline')->defaultValue('%spy_timeline.class.timeline%')->cannotBeEmpty()->end()
+                        // NEXT_MAJOR: Change this to App\Enity\User
```
Comment (greg0ire) [inline src/DependencyInjection/Configuration.php]: Why do I still see `Enity` here? Github display bug?

**Comment by greg0ire** [comment]
Thanks @core23 !

**Pull request merged**